### PR TITLE
A9: Float Layout

### PR DESCRIPTION
**Pull request by Brandonz07**
Completed

### CODE REVIEW

File: src/br/html/A9-Float-Layout.html [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+      rel="stylesheet"
+      href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/6.5.2/css/all.min.css"
+      integrity="sha512-SnH5WK+bZxgPHs44uWIX+LLJAJ9/2PkPKZ5QiAj6Ta86w+fsb2TkcmfRyVX3pBnMFcV7oQPJkl9QevSCWr3W6A=="
+      crossorigin="anonymous"
+      referrerpolicy="no-referrer"
+    />
+  </head>
+  <body>
+    <nav>
+      <div class="perfil">
```
Comment (luisalejandrosalazar) [inline src/br/html/A9-Float-Layout.html:20]: Avoid to create and use classes just for one component
Please refactor

File: src/br/html/A9-Float-Layout.html [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+      <a href="#">Christmas Tracks</a>
+      <a href="#">Other Remix</a>
+    </nav>
+    <main>
+      <h1>Universal Studio</h1>
+      <p>Connecting Paths...Come<br />With Us</p>
+
+      <div class="buttons">
+        <button class="btn-play">play <i class="fa-solid fa-play"></i></button>
+        <button class="btn-more">More info +</button>
```
Comment (luisalejandrosalazar) [inline src/br/html/A9-Float-Layout.html:46]: Avoid to create and use classes just for one component
Please refactor

File: src/br/html/styles/styleA9.css [hunk 3]
```diff
@@ -0,0 +1,96 @@
...
+  background-color: #201c1c;
+}
+
+nav a {
+  text-decoration: none;
+  color: white;
+  font-size: 22px;
+  display: block;
+  text-align: justify;
+  padding: 3%;
```
Comment (luisalejandrosalazar) [inline src/br/html/styles/styleA9.css:31]: It is this property necessary?

File: src/br/html/A9-Float-Layout.html [hunk 4]
```diff
@@ -0,0 +1,50 @@
...
+    />
+  </head>
+  <body>
+    <nav>
+      <div class="perfil">
+        <img class="photo" src="./img/perfil.jpg" alt="" />
+        <a href="">Ikako Talavazo</a>
+      </div>
+
+      <a href="#"><i class="fa-solid fa-house"></i>Home</a>
```
Comment (luisalejandrosalazar) [inline src/br/html/A9-Float-Layout.html:25]: A navigation bar is a list of links, please check this

https://www.w3schools.com/css/css_navbar.asp

Please refactor it using a list, and use more consistent margin and padding values.

**Review by luisalejandrosalazar — COMMENTED** [review]
Use a margin or padding like:
padding: 10% 0 10% 8%; 
margin: 10% 0 0 30%;

Are not a good practice..

Please refactor it using more consistent margin and padding values.

File: src/br/html/A9-Float-Layout.html (hunk 1 above)
Comment (Brandonz07) [reply src/br/html/A9-Float-Layout.html]: remove class

File: src/br/html/A9-Float-Layout.html (hunk 2 above)
Comment (Brandonz07) [reply src/br/html/A9-Float-Layout.html]: remove class and add pseudoclasses

File: src/br/html/styles/styleA9.css (hunk 3 above)
Comment (Brandonz07) [reply src/br/html/styles/styleA9.css]: no

File: src/br/html/A9-Float-Layout.html (hunk 4 above)
Comment (Brandonz07) [reply src/br/html/A9-Float-Layout.html]: add li

File: src/br/html/A9-Float-Layout.html [hunk 5]
```diff
@@ -0,0 +1,50 @@
...
+      <li><a href="#"><i class="fa-solid fa-house"></i>Home</a></li>
+      <li><a href="#"><i class="fa-solid fa-table-cells-large"></i>Category</a></li>
+      <li><a href="#"><i class="fa-solid fa-bullhorn"></i>Featured</a></li>
+      <li><a href="#"><i class="fa-solid fa-cart-shopping"></i>U.A Store</a></li>
+      <li><a href="#"><i class="fa-regular fa-clock"></i>History</a></li>
+      <li><a href="#"><i class="fa-regular fa-heart"></i>Favourities</a></li>
+      <li><a href="#"><i class="fa-regular fa-thumbs-up"></i>Liked Videos</a></li>
+      <li><a href="#"><i class="fa-regular fa-circle-play"></i>Subscriptions</a></li>
+      <li><a href="#"><i class="fa-solid fa-clapperboard"></i>Records</a></li>
+      <li><a href="#"><i class="fa-solid fa-folder-plus"></i>Create Playlist</a></li>
```
Comment (luisalejandrosalazar) [inline src/br/html/A9-Float-Layout.html:34]: li elements belong to a ul or ol element.
please refactor it

Comment (Brandonz07) [reply src/br/html/A9-Float-Layout.html]: ul add

**Review by luisalejandrosalazar — APPROVED** [review]
Approved

**Pull request merged**